vendor/k8s.io/kubernetes/pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -281,7 +281,7 @@ func (adc *attachDetachController) populateActualStateOfWorld() error {
 				glog.Errorf("Failed to mark the volume as attached: %v", err)
 				continue
 			}
-			adc.processVolumesInUse(nodeName, node.Status.VolumesInUse, true /* forceUnmount */)
```

Comment:
We use this to build `openshift start kubernetes controller-manager` which is supposed to be true to the upstream.  Unconditional carries like this shouldn't be added and we are actively trying to remove the ones we have.

Reply:
right I don't disagree, but we have hit bit of road block in merging this PR upstream.cc @eparis @childsb

Reply:
https://github.com/kubernetes/kubernetes/pull/52221

Reply:
@gnufied is not allowed to let 52221 miss for 1.9. But this must be carried in 3.7+ even if that doesn't happen. This is mission critical.

Reply:
> @gnufied is not allowed to let 52221 miss for 1.9. But this must be carried in 3.7+ even if that doesn't happen. This is mission critical.

This is currently listed as a carry.  As such, it will prevent convergence with upstream.  If its related to an upstream pull, then it doesn't cause us to diverge forever and its a risk assessment, not a case of preventing any future reconciliation.

Is this just an upstream pick?

Reply:
It is a pick, but if upstream fails to merge, it is a carry. you tell me   :)

Reply:
> It is a pick, but if upstream fails to merge, it is a carry. you tell me :)

Its a pick, reference the upstream number.  If you don't, we'll never know that we either have to never converge onto upstream or find another way to handle it.

Reply:
okay I am happy rewording the commit. doesn't seem like a biggie.

Reply:
Alright I have updated this PR to have upstream's PR ref. number.